src/templates/app/user/settings/personal_details.jsx

```diff
@@ -36,8 +41,14 @@ const PersonalDetails = () => (
 
         <form className='form gr-padding-10 invisible' id='frmPersonalDetails'>
             <Fieldset legend={it.L('Details')}>
+                <Salutation className='invisible' />
+                <FirstName className='invisible' />
+                <LastName className='invisible' />
+                <DateOfBirth className='invisible' />
+                <Citizenship className='invisible' />
+                <AccountOpeningReason className='invisible' />
```

Comment:
usually we put the label next to the input row so we know that both exist, e.g. look at citizenship below

Reply:
done